deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -457,7 +457,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                 currentCell: currentCluster.cellName,
                 hostTypeOptions: info.hostTypes.map(function(item, idx) {
                     return {
-                        value: item.abstract_name,
+                        value: item.id,
```

Comment:
@disparter how will UI look like with this change? I understand it will show IDs in dropdown instead of abstract_name/provider_name?

Ideal state: drop down should show -> Abstract name or Provider name or both

Reply:
Value is the field that is sent into the form
    <input value="">
And the text is the value that will be shown to user
So in the case is:
 text: item.abstract_name + " (" + item.core + " cores, " + item.mem + " GB, " + item.storage + ", " + item.provider + ": " + item.provider_name + ")",